### ISSUE TITLE
date for notes

### ISSUE THREAD

**Issue by GemperC** [Yarden-zmr/obsidian-zola-plus#12]

**Comment by orditeck** [comment]
@GemperC I don't have any helpful doc to provide but I implemented it in [my fork](https://github.com/orditeck/obsidian-zola-plus). It's using frontmatter from the `.md`, eg:

```
---
title: "You can also override the title"
created: 2023-02-01
updated: 2023-02-10
extra:
    something: "I'm also passing down any extra to the template, can be useful!"
---
```
Output when only `created` is provided:
```
Posted February 1, 2023
```

Output when both `created` and `updated` are provided (and are different):
```
Posted February 1, 2023, last updated February 10, 2023
```


I'll eventually document it but for now hopefully this can help you!

**Comment by orditeck** [comment]
Yeah I think that'd be a great option to have for sure. I personally prefer having control over those dates, especially the updated one. My garden is very recent and isn't read by anyone at the moment but I'd like to think that it will be eventually, and I'd rather not have the Last Updated date changed for a quick typo fix.
